frontend/src/app/app.routes.ts

```diff
@@ -316,3 +319,19 @@ export function taskRoute(taskListType: string): Route {
     runGuardsAndResolvers: 'paramsOrQueryParamsChange'
   };
 }
+
+/**
+ * Creates a persons route of a given type
+ * This function has no reason to be exported, other than to make the AOT compiler happy.
```

Comment:
just for my curiosity, could you explain me that : "to make the AOT compiler happy"

Reply:
When we test the app while developing, using ng serve, we use what Angular calls the *JIT compiler*: HTML templates are bundled as is in the JS and are parsed and compiled to JavaScript classes in the browser, when the application starts. That's why the "Loading" message takes a while to disappear. In production mode (i.e. when we actually build the app using gradle, which calls `ng build --prod`), we use the *AOT compiler*. It compiles the HTML templates to TypeScript, then the TypeScript to JS, and the result of this compilation is stored in the bundle. That makes the app start much faster, at the cost of a compilation time that is bigger. But it also imposes some bizarre restrictions on the code, and this is one of them.